### DISCUSSION TITLE
civil commnets subgroups training

### DISCUSSION THREAD

**Discussion by IbtihalFerwana** [p-lambda/wilds#100] [Q&A]
Hello, 
Thanks for making your library easy to navigate.

I'm trying to reproduce your results using the executable version here: https://worksheets.codalab.org/worksheets/0x7985440df1b742039badfacb4669d0d2 
Can you please elaborate for me what does IRM x black mean? specifically in the datasets configs: `groupby_fields': ['black', 'y']` what would that imply?

**Comment by kohpangwei** [comment]
Hi Ibtihal,

It means that we're forming groups based on the attributes `black` and `y`. Specifically, we form one group for each possible setting of those attributes, resulting in 4 groups since they are both binary attributes. We then run IRM using these four groups as "domains" (to be invariant over). Does that make sense?
